pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -431,6 +430,26 @@ def _flatten_object(parent, name):
                 parent['{}_{}'.format(name, key)] = value
             del parent[name]
 
+    @staticmethod
+    def _categorize_documents(document, legal_provisions, laws, hints):
+        """
+        Categorize document by their documentType (LegalProvision, Law or Hint)
+
+        Args:
+            dcoument (dict): The document type as dictionary.
+            legal_provisions (dict): The legal_provisions dictionary to fill.
+            laws (dict): The laws dictionary to fill.
+            hints (dict): The Hints dictionary to fill.
+        """
+        uid = Renderer._get_element_of_legal_provision_maybe_uid(document)
+        documentType = document['DocumentType']
+        if documentType == 'LegalProvision':
+            legal_provisions[uid] = document
+        elif documentType == 'Law':
+            laws[uid] = document
+        else:
+            hints[uid] = document
+
```

Comment:
Should cover the case if there is no DocumentType and not default to hint.

Reply:
if there is no DocumentType, we now get an exception. The default case is only if type is present, but not LegalProvision or Law